src/Sqids.php

```diff
@@ -621,22 +268,19 @@ public function __construct(
             );
         }
 
+        // Filter out blocklist words that are shorter than 3 characters or contain non-alphabet characters
```

Comment:
the condition about non-alphabet characters does not seem to exist anymore for the filtered list

Reply:
Thanks. I removed this comment.